source/content/glossar/bibliothek/dcat-examples/dataset-publisher-rdf.rst

```diff
@@ -1,5 +1,5 @@
 .. code-block:: xml
-    :caption: Publisher of a Dataset current implementation
+    :caption: Publisher of a dataset: current implementation in RDF
```

Comment:
It think we should remove the word "current" here: "Publisher of a dataset in RDF"

Reply:
I agree, removed it.